easybuild/easyconfigs/s/snakemake/snakemake-8.27.0-foss-2024a.eb

```diff
@@ -0,0 +1,153 @@
+easyblock = 'PythonBundle'
+
+name = 'snakemake'
+version = '8.27.0'
+
+homepage = 'https://snakemake.readthedocs.io'
+description = "The Snakemake workflow management system is a tool to create reproducible and scalable data analyses."
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+builddependencies = [
+    ('poetry', '1.8.3'),
+]
+dependencies = [
+    ('Python', '3.12.3'),
+    ('Python-bundle-PyPI', '2024.06'),
+    ('SciPy-bundle', '2024.05'),
+    ('GitPython', '3.1.43'),
+    ('IPython', '8.14.0'),
```

Comment:
8.28.0 is available, https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/i/IPython/IPython-8.28.0-GCCcore-13.3.0.eb